tutorials/get-started/index.md

```diff
@@ -0,0 +1,989 @@
+which displays "Hello World" in the center of the screen.
+
+<!-- skip -->
+{% prettify dart %}
+import 'package:flutter/material.dart';
+
+void main() => runApp(new MyApp());
+
+class MyApp extends StatelessWidget {
+  // This widget is the root of your application.
+  @override
+  Widget build(BuildContext context) {
+    return new MaterialApp(
+      title: 'Welcome to Flutter',
+      home: new Scaffold(
+        appBar: new AppBar(
+          title: new Text('Welcome to Flutter'),
+        ),
+        body: new Center(child: new Text('Hello World')),
+      ),
+    );
+  }
+}
+{% endprettify %}
+</li>
+
+<li markdown="1">
+Run the app. You should see the following screen.
+
+<center><img src="images/hello-world-screenshot.png" alt="screenshot of hello world app"></center>
+</li>
+
+</ol>
+
+### Observations
+
+<ul markdown="1">
+<li markdown="1"> This example creates a Material app.
+</li>
+<li markdown="1"> The main method specifies fat arrow (`=>`) notation,
+     which is short hand used for one-line functions or methods.
+</li>
+<li markdown="1"> The app extends StatelessWidget which makes the app itself a
+     widget. In Flutter, most everything is a widget, including
+     alignment, padding, and layout.
+</li>
+<li markdown="1"> The Scaffold widget, from the Material library,
+     provides a default app bar, title, and a body property that
+     holds the widget tree for the home screen. The widget subtree
+     can be quite complex.
+</li>
+<li markdown="1"> The widget tree for this example consists of a Center widget
+     containing a Text child widget. The Center widget aligns its
+     widget subtree to the center of the screen.
+</li>
+
+<li markdown="1"> A widget’s main job is to provide a `build` function
+    that describes how to display the widget in terms of other,
+    lower level widgets.
+</li>
+<li markdown="1"> Moving the “hello world” text into a separate widget,
+    HelloWorld, results in an identical widget tree as the code abovr.
+    (This code is informational only. You are starting with the Hello
+    World code above.)
+
+<!-- skip -->
+{% prettify dart %}
+import 'package: flutter/material.dart';
+
+class MyApp extends StatelessWidget {
+ @override
+ Widget build(BuildContext context) {
+   return new MaterialApp(
+     title: 'Welcome to Flutter',
+     home: new Scaffold(
+       appBar: new AppBar(
+         title: new Text('Welcome to Flutter'),
+       ),
+       body: new Center(
+         child: new Text('Hello World')
+       ),
+     ),
+   );
+ }
+}
+{% endprettify %}
+</li>
+</ul>
+
+---
+
+## Step 2: Use an external package
+
+In this step, you’ll start using an open-source package named
+**english_words**, which contains a few thousand of the most used
+English words plus some utility functions.
+
+You can find the
+[english_words](https://pub.dartlang.org/packages/english_words)
+package, as well as many other open source packages, on
+[pub.dartlang.org](https://pub.dartlang.org/).
```

Comment:
I suggest we for the second link use `https://pub.dartlang.org/flutter` to get the flutter landing page and not the general front page.

Reply:
Done